docs/src/Dropdown.doc.js

```diff
@@ -160,58 +147,52 @@ card(
       {
         name: 'badgeText',
         type: 'string',
-        description: "When supplied, will display a Badge next to the item's label.",
-        href: 'default',
+        description:
+          "When supplied, will display a Badge next to the item's label. See the [Badges](#Badges) variant to learn more.",
       },
       {
         name: 'children',
         type: 'React.Node',
         description:
-          'If needed, users can supply custom content to each Dropdown Item. This can be useful when extra functionality is needed beyond a basic Link, such as supporting React Router.',
-        href: 'withCustomItemContent',
+          'If needed, users can supply custom content to each Dropdown Item. This can be useful when extra functionality is needed beyond a basic Link. See the [Custom item content](#Custom-item-content) variant to learn more.',
       },
       {
         name: 'isExternal',
         type: 'boolean',
         description:
-          'When true, adds an arrow icon to the end of the item to signal this item takes users to an external source. Do not add if the item navigates users within the app.',
-        href: 'default',
+          'When true, adds an arrow icon to the end of the item to signal this item takes users to an external source. Do not add if the item navigates users within the app. See the [Best practices](#Best-practices) for more info.',
       },
       {
         name: 'option',
         type: '{| label: string, value: string, subtext?: string |}',
         required: true,
         description: 'Object detailing the label, value, and (optional) subtext for this item.',
```

Comment:
is optional redundant based on type definition?

Reply:
I think for those who don't know typing, it's still helpful